src/main/java/io/airlift/slice/OutputStreamSliceOutput.java

```diff
@@ -62,6 +66,12 @@ public int size()
     }
 
     @Override
+    public int getRetainedSize()
+    {
+        return INSTANCE_SIZE;
```

Comment:
Maybe add a javadoc comment that this does not include the size of any buffers retailed by the nested output stream